doc/source/python-api.rst

```diff
@@ -18,86 +18,17 @@ Features:
 * Enables out-of-core access to data aggregations much larger than single-host main memory
 * Enables distributed computation over datasets
 
-Modules
```

Comment:
This is an iterative improvement but not entirely right. @ebezzi points out this is not inherting methods from `somacore`. This is not a final product. I can omit the changes on this particular from the current PR if that helps it get accepted/merged sooner.